employee-rostering-shared/src/main/java/org/optaweb/employeerostering/shared/roster/RosterState.java

```diff
@@ -177,14 +179,11 @@ public void setDraftLength(Integer draftLength) {
         this.draftLength = draftLength;
     }
 
+    @JsonIgnore
     public Integer getPublishLength() {
         return publishLength;
     }
 
-    public void setPublishLength(Integer publishLength) {
```

Comment:
Is the RosterState ever send to the server? I wasn't sure if jackson doesn't need the setter to unmarshal it?

Reply:
When you create a Tenant, you need to supply the initial `RosterState`. I added an `@JsonIgnore` on the getter since publish length is not sent (since it is non-modifiable)